### ISSUE TITLE
High storage use

### ISSUE THREAD

**Issue by hjek** [abdallahmehiz/mpvKt#93]
**Describe the bug**
mpvKt uses a lot of user data, >300 MB.

**To Reproduce**
1. Watch some films
2. View mpvKt settings
3. It's using a lot of user data.

**Expected behavior**
I'm fine with mpvKt using a lot of disk space temporarily, but I'd like to be able to clear these temporary files without deleting the app settings.

**Actual behavior**
If I clear the >300MB of user data to reclaim disk space, then I also delete the app settings.

**Screenshots**
![Screenshot_20240918-190253](https://github.com/user-attachments/assets/d2b3c69c-d069-4cc1-b0d0-44235e8f4973)

**Additional info (please complete the following information):**
 - Device: moto e13
 - Android version: 12
 - App Version: v0.1.5 (from F-Droid)

**Additional context**
Thanks for making awesome Android interface for mpv.

**Comment by abdallahmehiz** [comment]
Have you set an mpv configuration directory in advanced settings? cause files that are placed there will get copied to the app's data directory.
You can clear some mpv configuration and font cache from the settings. Try doing that and checking the size again.

**Comment by abdallahmehiz** [comment]
> Oh, so if I set `~/Documents` as configuration directory, mpvKt will copy all contents of that dir into its private storage?

Yes it'll copy the entire directory's contents. I've made it this way to allow copying scripts.

I'll consider this issue closed then

**Comment by abdallahmehiz** [comment]
Well that's a bug...
You can mitigate it by going into advanced settings and saving a blank mpv.conf and input.conf
